builtin/providers/cloudstack/resource_cloudstack_vpc.go

```diff
@@ -152,6 +157,16 @@ func resourceCloudStackVPCRead(d *schema.ResourceData, meta interface{}) error {
 	setValueOrUUID(d, "project", v.Project, v.Projectid)
 	setValueOrUUID(d, "zone", v.Zonename, v.Zoneid)
 
+	// Grab the source NAT IP that CloudStack assigned.
+	p := cs.Address.NewListPublicIpAddressesParams()
+	p.SetVpcid(d.Id())
+	p.SetIssourcenat(true)
+	p.SetProjectid(v.Projectid)
+	l, e := cs.Address.ListPublicIpAddresses(p)
```

Comment:
Very (very) small nit, but errors are called `err` all over TF. So seems more inline with the existing code to also do that here :wink: